src/WinRT.Runtime/ActivationFactory.cs

```diff
@@ -0,0 +1,272 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using WinRT.Interop;
+
+namespace WinRT
+{
+    internal unsafe sealed class DllModule
+    {
+        readonly string _fileName;
+        readonly IntPtr _moduleHandle;
+        readonly delegate* unmanaged[Stdcall]<IntPtr, IntPtr*, int> _GetActivationFactory;

```

Comment:
Nit: don't use `IntPtr` instead of actual pointers, when you are in fact dealing with pointers.
```suggestion
        readonly delegate* unmanaged[Stdcall]<void*, void**, int> _GetActivationFactory;

```

Reply:
Keeping as is as we use IntPtr everywhere and this would result in much more changes unrelated to moving this into WinRT.Runtime.